app/schema.graphql

```diff
@@ -65,6 +65,17 @@ type Dataset {
   events(eventIds: [ID!]!, dimensions: [DimensionInput!]): [Event!]!
 }
 
+type DatasetInfo {
+  """The start bookend of the data"""
+  startTime: DateTime!
+
+  """The end bookend of the data"""
+  endTime: DateTime!
+
+  """The record count of the data"""
+  recordCount: Int!
+}
```

Comment:
Not needed but this could be an interface that the "ModelDataset" extend from - just if we wanted the schema to be a tad more concise.